vault-plugins/flant_iam_auth/io/kafka_handlers/root/restore.go

```diff
@@ -25,7 +26,33 @@ func HandleRestoreMessagesRootSource(txn *memdb.Txn, objType string, data []byte
 	case iam.TenantType:
 		inputObject = &iam.Project{}
 		table = iam.TenantType
-
+	case iam.FeatureFlagType:
+		inputObject = &iam.FeatureFlag{}
+		table = iam.FeatureFlagType
+	case iam.GroupType:
+		inputObject = &iam.Group{}
+		table = iam.GroupType
+	case iam.RoleType:
+		inputObject = &iam.Role{}
+		table = iam.RoleType
+	case iam.RoleBindingType:
+		inputObject = &iam.RoleBinding{}
+		table = iam.RoleBindingType
+	case iam.RoleBindingApprovalType:
+		inputObject = &iam.RoleBindingApproval{}
+		table = iam.RoleBindingApprovalType
+	case iam.MultipassType:
+		inputObject = &iam.Multipass{}
+		table = iam.MultipassType
+	case iam.ServiceAccountPasswordType:
+		inputObject = &iam.ServiceAccountPassword{}
+		table = iam.ServiceAccountPasswordType
+	case iam.IdentitySharingType:
+		inputObject = &iam.IdentitySharing{}
+		table = iam.IdentitySharingType
+	case model.ServerType:
+		inputObject = &model.Server{}
+		table = model.ServerType
```

Comment:
Repositories have `Sync` method for this. These methods hide unmarhalling and the decision to update or delete an objects. Should be used here. Maybe not right in this PR.

Reply:
not in this pr